moduleroot/Rakefile

```diff
@@ -7,3 +7,27 @@ PuppetLint.configuration.fail_on_warnings = true
 PuppetLint.configuration.send('<%= check %>')
 <% end -%>
 PuppetLint.configuration.ignore_paths = ["spec/**/*.pp", "pkg/**/*.pp"]
+
+desc 'Generate pooler nodesets'
+task :gen_nodeset do
```

Comment:
I think moving it into PSH would be great if beaker-hostgenerator had vagrant support and was generally applicable beyond Puppet's infra. But it doesn't so I don't think, at this time, PSH is the best place for it. However, when vagrant support is added to beaker-hostgenerator, most of this code could easily be moved into PSH.